Sources/ConcentricOnboarding/ConcentricOnboardingView.swift

```diff
@@ -28,57 +28,59 @@ class ObservableInt: ObservableObject {
             didSet?()
         }
     }
-
+    
     var didSet: (()->())?
-
+    
     init(_ value: Int = 0) {
         self.value = value
     }
 }
 
 public struct ConcentricOnboardingView : View {
-
+    
     public var animationWillBegin = {}
     public var animationDidEnd = {}
     public var didGoToLastPage = {}
     public var insteadOfCyclingToFirstPage: (()->())? // replaces default navigation to first page after pressing next on last page
     public var insteadOfCyclingToLastPage: (()->())? // replaces default navigation to last page after pressing prev on first page while navigating backwards
     public var didPressNextButton: (()->())? // replaces default button action with user's custom closure
-
+    
     public var currentPageIndex: Int {
         return currentIndex.value
     }
-
+    
     let radius: Double = 30
     let limit: Double = 15
-
+    
     static let timerStep = 0.02
     let timer = Timer.publish(every: timerStep, on: .current, in: .common).autoconnect()
     var step: Double {
         2*limit / (duration / ConcentricOnboardingView.timerStep)
     }
-
+    
     let pages: [AnyView]
     let bgColors: [Color]
     let duration: Double // in seconds
-
+    
     @ObservedObject var currentIndex = ObservableInt()
     @ObservedObject var nextIndex = ObservableInt(1)
     @ObservedObject var isAnimating = ObservableBool()
     @ObservedObject var isAnimatingForward = ObservableBool(true)
-
+    
     @State var progress: Double = 0
     @State var bgColor = Color.white
     @State var circleColor = Color.white
-
+    let nextIcon: String // the default icon is the cupertino next Icon button which is "chevron.forward but it can be change from the prop.
```

Comment:
I see you didn't fix the " problem, and I noticed a typo, could you please user this line?
// the default icon is "chevron.forward", use constructor argument to change

Reply:
the default arrow is pointing forward... that was just a demo that the user can use any icon of their choice...